src/implementations/vanilla/components/table/table.scss

```diff
@@ -73,6 +73,14 @@ $table: null !default;
   }
 }
 
+.ecl-table__caption {
+  caption-side: bottom;
```

Comment:
considering that most likely we will receive requests to make this position parametrical, why don't we add this option already?

Reply:
The position has been decided by the designers.
So, sure, we could make it as a parameter, but this may be just an option never used. I'd rather wait to have a use case